dockerfiles/postgres16-container-disk:c10s/install_postgres.sh

```
@@ -0,0 +1,15 @@
+#Pod image: quay.io/sclorg/postgresql-16-c10s
+Code: https://github.com/sclorg/postgresql-container
+https://linuxize.com/post/how-to-install-postgresql-on-centos-8/
```

Shellcheck (0.11.0)

[error] This is interpreted as a command name ending with '/'. Double check syntax.

(SC2287)